### PR TITLE
Amjith/changelog 0.19.0

### PR DESCRIPTION
**Pull request by amjith**
We're getting ready to do another release. 

@dbcli/pgcli-core Can you please check the changelog to see if I've missed anything? 

Give it a thumbs up if you're satisfied. I'll merge it in once I get a quorum.

### CODE REVIEW

**Comment by j-bennet** [comment]
:+1:

File: changelog.rst [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+  history_file = /path/to/history/file
+
+* Add support for running queries from a file using ``\i`` special command. (Thanks: `Michael Kaminsky`_)
+
+BugFixes:
+---------
+
+* Always use utf-8 for database encoding regardless of the default encoding used by the database.
+* Fix for None dereference on ``\d schemaname.`` with sequence. (Thanks: `Nathan Jhaveri`_)
+* Fix a crashing bug in the autocompletion enginer for some ``JOIN`` queries.
```
Comment (darikg) [inline changelog.rst]: enginer -> engine

**Comment by darikg** [comment]
:+1:

**Comment by amjith** [comment]
Thanks for catching the typo. 

Merging now.

**Pull request merged**